content/en/docs/usage/import-clusters.md

```diff
@@ -34,7 +34,7 @@ One of the easiest ways to connect and authenticate to a cluster is to use the k
 
 First you need to base64 encode the kube config for the imported cluster.
 ```bash
-base64 ./kubeconfig
+cat ~/.kube/config | base64 -w 0
```

Comment:
`~/.kube/config` is not used here because it may be very large, and *. /kubeconfig* is the configuration that represents a particular cluster.

The `base64 <file>` doesn't work on mac, thanks for catching that!

But on mac, `base64` doesn't support the `-w` flag, it will not print newlines by default, so there is no need for the `-w` parameter either.

I think it can be changed here to look like this
```bash
# mac
cat ./kubeconfig | base64

# linux
cat ./kubeconfig | base64 -w 0
```

Reply:
Got it. I will change it again